source/_integrations/flipr.markdown

```diff
@@ -39,6 +35,39 @@ You will need to use the standalone app for this device to register a username a
 
 {% include integrations/config_flow.md %}
 
+## Flipr
+
+Flipr sends data like pH, chlorine or temperature to a cloud server on a regular basis in order to monitor your pool.
+
+There is currently support for the following information within Home Assistant via **sensors** and **binary_sensors** :
+
+## Entities
+
+### Sensors
+
+- Chlorine : the chlorine level
+- pH : the pH level of the water
+- Water Temperature : the temperature of the water
```

Comment:
```suggestion
- Water temperature : the temperature of the water
```
we use (sentence-style capitalization)[https://learn.microsoft.com/en-us/style-guide/capitalization]